cleanowners.py

```diff
@@ -85,10 +85,11 @@ def main():  # pragma: no cover
         usernames = get_usernames_from_codeowners(codeowners_file_contents)
 
         for username in usernames:
+            org = organization if organization else repo.owner.login
             # Check to see if the username is a member of the organization
-            if not github_connection.organization(organization).is_member(username):
+            if not github_connection.organization(org).is_member(username):
```

Comment:
Will this line of code work if org is set to a username? I thought you could only feed github_connection.organization() a proper org name.

Reply:
That's a good question.  Let me test that locally real quick.

Reply:
@zkoppert you are correct.  This fails for user repositories.  We need to check if the `org` is an actual org. Looking for best API call for that.  Any suggestions?

Reply:
@zkoppert pushed up change.  This will validate both the `ORGANIZATION` env var is an org and any org prefixes on repositories in `REPOSITORY` env var. wdyt?